### ISSUE TITLE
Main navigation is not accessible

### ISSUE THREAD

**Issue by Angelmmiguel** [bitnami/hex#192]
The submenus of the main navigation are not accessible. When you press enter nothing happens. We need to trigger the submenu open with the keyboard
